extensions.md

```diff
@@ -30,19 +32,30 @@ the less breaking changes of the next level.
-| [Fields](fragments/fields/README.md)                             | *None*         | <http://stacspec.org/spec/api/1.0.0-beta.1/req/fields>          | Adds parameter to control which fields are returned in the response.                                                                    | *Pilot*    |
-| [Query](fragments/query/README.md)                               | *None*         | <http://stacspec.org/spec/api/1.0.0-beta.1/req/query>           | Adds parameter to search Item and Collection properties.                                                                                | *Pilot*    |
-| [Context](fragments/context/README.md)                           | [ItemCollection](core/itemcollection-spec.md) | <http://stacspec.org/spec/api/1.0.0-beta.1/req/context>         | Adds search related metadata (context) to ItemCollection.                                                                               | *Proposal* |
-| [Sort](fragments/sort/README.md)                                 | *None*         | <http://stacspec.org/spec/api/1.0.0-beta.1/req/sort>            | Adds Parameter to control sorting of returns results.                                                                                   | *Pilot*    |
-| [Transaction](ogcapi-features/extensions/transaction/README.md)                   | *None*         | <http://www.opengis.net/spec/ogcapi-features-4/1.0/conf/simpletx> | Adds PUT and DELETE endpoints for the creation, editing, and deleting of items and Collections.                                         | *Pilot*    |
-| [Items and Collections API Version](ogcapi-features/extensions/version/README.md) | *None*         | <http://stacspec.org/spec/api/1.0.0-beta.1/req/version>         | Adds GET versions resource to collections and items endpoints and provides semantics for a versioning scheme for collections and items. | *Proposal* |
+**NOTE**: *Currently the fragments are only used in item-search, but in the next release we will define extensions for all the fragments that
+are scoped against ogcapi-features*.
+
+## List of community extensions
 
-\* The scope refers to the STAC specifications an extension extends. As all extensions here are API extensions,
-the API is not mentioned explicitly as scope and only the core STAC specifications (Catalog, Collection, Item and ItemCollection) are listed.
+| Extension Name                                                                    | Scope                                                                                                     | Fragment                               | Conformance URI                                                            | Description                                                                                                                             | Maturity   |
```

Comment:
The table seems to contain to many details and is a bit overwhelming. From my perspective Fragments should be its own table (in fragments/README.md ?) and Conformance URI seems to be a implementation detail.

Reply:
Yeah, all seemed useful, but I can see how together its a bit much. I'm most inclined to remove fragments, since people will be able to find that pretty easily if they go to an extension, and I'm pretty sure there is a table in the fragments readme.

Conformance URI seems useful to me to have in the table, so if you're looking at a conformsTo document on a service you can easily go to this location and find it.

Reply:
For your use case, wouldn't you just load up the conformance URI and then get the OpenAPI?

I'm not sure, but would you think it's better two have two tables?

Extensions with Name, Scope, Description and Maturity 
and (in overview.md?)
Conformance URI -> Name (for all conformance URIs)

The table above is missing conformance URIs (e.g. core) and is not complete anyway...

Reply:
Two tables sounds fine to me. I'll update it now.